rts/Lua/LuaVBOImpl.cpp

```diff
@@ -712,6 +722,14 @@ sol::as_table_t<std::vector<lua_Number>> LuaVBOImpl::Download(sol::optional<int>
 	return sol::as_table(dataVec);
 }
 
+void LuaVBOImpl::Clear()
+{
+	VBOExistenceCheck(vbo, __func__);
+
+	GLubyte val = 0;
```

Comment:
> Perhaps clearing value should be a parameter with default of 0 in case it's unspecified?

Thought about that, but perhaps the complication of code isn't worth the nicety, unless we get a real demand for it. To me, the need to fill a buffer with repeated number which is not a zero feels exotic.
(A small complication of code is in that the Clear function will have to discern the value format and wire GL parameters based off that, compared to simple operation on bytes)

Reply:
Yeah maybe